### DISCUSSION TITLE
Showing all 458 wallets.

### DISCUSSION THREAD

**Discussion by PXLFuSSeL** [reown-com/appkit#3317] [General]
Good evening, 

Just a question for my understanding, how wallets got selected ot be shown?
for example 5thscape.com got 450+ shown and on my dapp are just 40+ show.
Whats the reason behind that?

greetings

**Comment by arein** [comment]
gm - what's your app?

It depends on the network - not all wallets support all networks. However, we are overhauling this currently as it can cause some confusion. In the first step wallets that support adding custom networks will show up. This should be live next week.
